test/coreneuron/test_spikes.py

```diff
@@ -125,6 +125,7 @@ def run(mode):
             run(mode)
         # Make sure that file mode also works with custom coreneuron.data_path
         if file_mode:
+            coreneuron.file_mode = True
```

Comment:
@WeinaJi : I didn't verify but `with coreneuron(enable=True, gpu=enable_gpu, file_mode=file_mode, verbose=0)` not doing that initialisation?

Reply:
You are right, I reverted that change.